db/migrate/20170203113625_remove_organisation_reference_from_content_items.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveOrganisationReferenceFromContentItems < ActiveRecord::Migration[5.0]
+  def change
+    remove_reference :content_items, :organisation, foreign_key: true
```

Comment:
I have addressed the comment.. I was only recall our previous conversatioin as the reason why this was omitted.